nilearn/datasets/atlas.py

```diff
@@ -20,6 +20,103 @@
 _TALAIRACH_LEVELS = ['hemisphere', 'lobe', 'gyrus', 'tissue', 'ba']
 
 
+def fetch_difumo(dimension=64, resolution_mm=2, data_dir=None, resume=True, verbose=1):
+    """Fetch DiFuMo brain atlas
+
+    Direct download links from OSF:
+
+    dic = {64: https://osf.io/wjum7/download,
+           128: https://osf.io/kdfg3/download,
+           256: https://osf.io/vza2y/download,
+           512: https://osf.io/a23gw/download,
+           1024: https://osf.io/jpdum/download,
+           }
+
+    Parameters
+    ----------
+    dimension : int
+        Number of dimensions in the dictionary. Valid resolutions
+        available are {64, 128, 256, 512, 1024}.
+
+    resolution_mm : int
+        The resolution in mm of the atlas to fetch. Valid options
+        available are {2, 3}.
+
+    data_dir : string, optional
+        Path where data should be downloaded. By default,
+        files are downloaded in home directory.
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        Dictionary-like object, the interest attributes are :
+
+        - 'maps': str, 4D path to nifti file containing regions definition.
+        - 'labels': string list containing the labels of the regions.
+
+    References
+    ----------
+    Dadi, K., Varoquaux, G., Machlouzarides-Shalit, A., Gorgolewski, KJ.,
+    Wassermann, D., Thirion, B., Mensch, A.
+    Fine-grain atlases of functional modes for fMRI analysis,
+    Paper in preparation
```

Comment:
You can remove this line. I would prefer the citation as denoted in this [link](https://github.com/Parietal-INRIA/DiFuMo)